erpnext/manufacturing/doctype/bom/bom.py

```diff
@@ -318,10 +318,11 @@ def validate_main_item(self):
 		if not item:
 			frappe.throw(_("Item {0} does not exist in the system or has expired").format(self.item))
 		else:
-			ret = frappe.db.get_value("Item", self.item, ["description", "stock_uom", "item_name"])
+			ret = frappe.db.get_value("Item", self.item, ["description", "stock_uom", "item_name", "inspection_required_before_manufacturing"])
 			self.description = ret[0]
 			self.uom = ret[1]
 			self.item_name= ret[2]
+			self.inspection_required = ret[3]
```

Comment:
This is overriding the user's selection in the BOM if they decide to uncheck it

Reply:
then should I removed that 
because as discuss with @ArundhatiSh  when it is checked it item master. then it has to be set

Reply:
Yeah, just show that behaviour to her

Reply:
@Alchez Just checked with Arundhati. Yup someone can uncheck this. so removing this